### ISSUE TITLE
data test silently ignored

### ISSUE THREAD

**Issue by zeitlinger** [kotest/kotest#2153]
**Which version of Kotest are you using**

4.4.2

```kotlin
class SpielTest : FreeSpec({

  "zum Zielpunkt laufen" { //if I change to  "zum Zielpunkt laufen" - { then it fails
        data class TestFall(val start: Punkt, val ziel: Punkt)

        forAll(
            TestFall(
                start = Punkt(x = 1000.0, y = 0.0),
                ziel = Punkt(x = 999.6464466094068, y = 0.35355339059327373)
            ),
            TestFall(start = Punkt(x = 0.0, y = 1000.0), ziel = Punkt(x = 0.0, y = 1000.0)),
            TestFall(start = Punkt(x = 0.0, y = 999.0), ziel = Punkt(x = 0.0, y = 999.5)),
            TestFall(start = Punkt(x = 0.0, y = 999.999), ziel = Punkt(x = 0.0, y = 1000.0))
        ) { testfall -> 1 shouldBe 2 }
})
```

**Comment by sksamuel** [comment]
Can you include the punkt class too.

**Comment by sksamuel** [comment]
I guess Punkt is just `data class Punkt(val x: Double, val y: Double)`

**Comment by sksamuel** [comment]
Ok, you cannot dynamically add root tests. Root tests must be added by the constructor only. This is confusing, and the subject of another ticket https://github.com/kotest/kotest/issues/2078

In the meantime, you can add either add a minus after your test name as you put in the comments, or have no test name at all.

This will be fixed for 4.5

**Comment by zeitlinger** [comment]
Great - then I can close this
